content/community/jdev-rules.md

```diff
@@ -0,0 +1,83 @@
+---
+Title: jdev Channel
+Url: community/channels/jdev
+---
+
+## About the XMPP Developers Channel ("jdev")
+
+The XMPP Developers Channel is located at [jdev@muc.xmpp.org](xmpp:jdev@muc.xmpp.org?join). It is a place primarily for developers of software following or aiming to follow XMPP standards (RFC 6120--6122, as well as XMPP Extension Proposals, and updated versions of the RFCs), to have civil and low-bar knowledge transfer, learning and support in the task of implementing the various pieces of the XMPP standard compendium.
+
+## Code of Conduct
+
+Like any growing community, the XMPP Developers Channel has a set of rules to govern the interactions, in order to make as many people as possible feel safe and welcome.
+
+Above all, the [Community Code of Conduct of the XMPP Standards Foundation](https://xmpp.org/extensions/xep-0458.html) (still under discussion, but unless stated otherwise in *this* document, the current experimental version applies nontheless) applies, called CoC hereafter. That means that behaviour which goes against the CoC is automatically not accepted.
+
+For transgressions solely happening within the XMPP Developers channel, the channel admins are an acceptable first point of contact.
+
+**Note**: The CoC only applies to XMPP Standards Foundation (XSF) members and venues. It does not automatically apply to all members of all XMPP channels. Do not report misconduct in venues outside of the control of the XSF to this room or the XSF Conduct Team, unless an XSF member is involved.
+
+## On-Topicness
+
+The channel started out as a place for developers to work together to improve the state of XMPP software in general, encompassing topics such as software interoperability (as opposed to deployment interoperability; for the latter, the [Operators channel](operators-rules.md) is the better a place), reliability, security and standards edge cases. It can provide valuable information when just reading passively, but also when actively asking.
```

Comment:
```suggestion
The channel started out as a place for developers to work together to improve the state of XMPP software in general, encompassing topics such as software interoperability (as opposed to deployment interoperability; for the latter, the [Operators channel](/community/channels/operators) is a better place), reliability, security and standards edge cases. It can provide valuable information when just reading passively, but also when actively asking.
```